series/series.go

```diff
@@ -190,6 +190,9 @@ func Bools(values interface{}) Series {
 	return New(values, Bool, "")
 }
 
+// Elements returns s.elements
+func (s Series) Elements() []Element { return s.elements }
```

Comment:
I'm not convinced that exporting all internal elements of a `Series` is a good idea for this particular use case. I actually would like to remove the existing access to `series.Element` but before doing that I have to review some of the functions that are using them right now.

Following my comment from the `dataframe.LoadStruct` method, this should then be removed.